website/docs/releases/2023/v2023.5.md

```diff
@@ -39,7 +39,11 @@ This release does not introduce any new requirements.
 
 ### docker-compose
 
-Download the docker-compose file for 2023.5 from [here](https://goauthentik.io/version/2023.5/docker-compose.yml). Afterwards, simply run `docker-compose up -d`.
+Download the docker-compose file for 2023.5 from [here](https://goauthentik.io/version/2023.5/docker-compose.yml). Afterwards, run `docker-compose up -d`.
+
+To download the latest `docker-compose.yml` file you can use the `wget` command followed by the file's location URL. The `-O` flag retains the downloaded file's name, overwriting any existing local file with the same name.
```

Comment:
I'd keep it simpler tbh:

To upgrade, download the new docker-compose file and update the docker stack with the new version, using these commands:

```shell
wget -O docker-compose.yml https://goauthentik.io/version/2023.5/docker-compose.yml
docker-compose up -d
```

(Not using a suggestion block since that breaks markdown a bit)

Reply:
Yeah, thanks, I like providing the actual commands... wasn't sure if most people will use `wget`.